lib/dart/lib/src/serializer/t_deserializer.dart

```diff
@@ -0,0 +1,41 @@
+/// Licensed to the Apache Software Foundation (ASF) under one
+/// or more contributor license agreements. See the NOTICE file
+/// distributed with this work for additional information
+/// regarding copyright ownership. The ASF licenses this file
+/// to you under the Apache License, Version 2.0 (the
+/// "License"); you may not use this file except in compliance
+/// with the License. You may obtain a copy of the License at
+///
+/// http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing,
+/// software distributed under the License is distributed on an
+/// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+/// KIND, either express or implied. See the License for the
+/// specific language governing permissions and limitations
+/// under the License.
+
+part of thrift;
+
+class TDeserializer {
+  final message = new TMessage('Deserializer', TMessageType.ONEWAY, 1);
+  TBufferedTransport transport;
+  TProtocol protocol;
+
+  TDeserializer() {
+    this.transport = new TBufferedTransport();
+    this.protocol = new TBinaryProtocolFactory().getProtocol(this.transport);
+  }
+
+  void read(TBase msg, Uint8List data) {
+    transport.setReadBuffer(data);
```

Comment:
Instead of changing the visibility of `setReadBuffer`, I think you can use the existing public method:
```
transport.writeAll(data);
```